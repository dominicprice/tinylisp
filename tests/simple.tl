1
(c 1 ())
(d x 42)
(s x 2)