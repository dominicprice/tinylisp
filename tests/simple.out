1
(1)
x
40